content/en/docs/glossary.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Glossary
+description: A list of Notary Project's terminologies 
+weight: 11
+---
+### Notary 
+Notary is a TUF [(The Update Framework)](https://github.com/theupdateframework/specification) based artifact signing solution. It consists of both [server](https://github.com/notaryproject/notary/blob/master/cmd/notary-server) and [client](https://github.com/notaryproject/notary/blob/master/cmd/notary) components for running and interacting with trusted collections. The client component is referred to as the Notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component.
+### Notation
+Notation is a non-TUF-based artifact signing solution. It refers to the implementation of the Notation CLI and the Notation Go libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with 3rd party key management solutions through a plugin model. The CLI itself is referred to as the "Notation CLI," while the libraries are referrred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. These libraries can be used independently of the Notation CLI.
+### Notary Project Specifications
+Notary Project Specifications are a collection of specifications distributed across "subprojects" within the Notary Project. These specifications are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary project tooling.
+
+Notary Project specifications are categorized into:
+
+**OCI Signature Specifications:**
+These specifications are specific to the Open Container Initiative (OCI). It describes how the Notary Project's Signing Scheme is applied to signatures stored in OCI registries. This category includes the following specifications:
+- [Notary Project OCI Signature Specification](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md)
```

Comment:
The `main` branch is a development branch so the content may be changed anytime. These URLs should be pointed to a specific tag (i.e. v1.0.0) after the `notaryproject` released v1.0.0 as described in https://github.com/notaryproject/notaryproject/issues/261.

Reply:
Acknowledged, I'll ensure to point the URLs to the specific tag (v1.0.0) after the `notaryproject` releases  v1.0.0.